tests/integration/targets/test_troubleshoot_rds_connectivity/tasks/delete_ec2.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Delete ec2 instance {{ ec2_instance_id }}
+  amazon.aws.ec2_instance:
+    instance_ids:
+      - "{{ ec2_instance_id }}"
+    wait: true
+    state: terminated
+  ignore_errors: true
+
+- name: Delete VM Key pair
+  amazon.aws.ec2_key:
+    name: "{{ ec2_key_name }}"
+    state: absent
+  ignore_errors: true
+
+- name: Delete security groups
+  amazon.aws.ec2_security_group:
+    state: absent
+    group_id: "{{ ec2_group_id }}"
+  ignore_errors: true
+
+- name: Delete internet gateway
+  amazon.aws.ec2_vpc_igw:
+    vpc_id: "{{ ec2_vpc_id }}"
+    state: absent
+  ignore_errors: true
+
+- name: Delete Subnet
+  amazon.aws.ec2_vpc_subnet:
+    cidr: "{{ ec2_subnet_cidr }}"
+    state: absent
+    vpc_id: "{{ ec2_vpc_id }}"
+  ignore_errors: true
+
+- name: Delete VPC
```

Comment:
@abikouo do you know why the integration test run is showing a dependency violation at this step? I know the terminator will eventually run and delete all the resources but it would be nice if this one was deleted during the test as expected.

Reply:
@hakbailey I am going to investigate

Reply:
@hakbailey the issue has been fixed. A route table deletion was missing.